lib/github.js

```diff
@@ -147,8 +147,8 @@ github.prototype = {
 							return false;
 
 
-						if(b.state == "APPROVED")
-							return true;
+                        if(b.state == "APPROVED" || b.state == "CHANGES_REQUESTED")
+                            return true;
 
 						if(b.submitted_at < pullRequest.updated_at)
```

Comment:
@Ahmadposten if line 153 is intended to not notify you if the pull request is in "Change requested" status and the author of the PR hasn't addressed the request, it's not working properly. Engineers on my team were getting notifications on PRs they had been requested to review even after requesting changes of the PR author. 

Because there's no consistent way to guarantee that a PR author addressed the changes requested reviewer (ex: on my team people usually @ the reviewer on the review comment thread when they've pushed a change to address the reviewer's feedback, but that might not be the case on other teams), I might just close this PR.